kitchen-tests/cookbooks/end_to_end/recipes/windows.rb

```diff
@@ -38,10 +38,39 @@
   action :enable
 end
 
+windows_user_privilege "BUILTIN\\Administrators" do
+  privilege %w{SeAssignPrimaryTokenPrivilege SeIncreaseQuotaPrivilege}
+  action :add
+end
+
 windows_firewall_profile "Public" do
   action :disable
 end
 
+windows_audit_policy "Update Some Advanced Audit Policies to Success and Failure" do
+  subcategory subcategory ["Application Generated", "Application Group Management", "Audit Policy Change"]
```

Comment:
```suggestion
  subcategory ["Application Generated", "Application Group Management", "Audit Policy Change"]
```

I suspect that all of these need to change and might be the cause of some of the CI failures.